Makefile

```diff
@@ -271,7 +271,7 @@ JOB ?= .*
 run-e2e: e2e-essentials ## Run e2e testing. JOB is an optional REGEXP to select certainn test cases to run. e.g. JOB=PR-Blocking, JOB=Conformance
 	kubectl apply -f cloud-config.yaml && \
 	cd test/e2e && \
-	$(REPO_ROOT)/$(GINKGO_V1) -v -trace -tags=e2e -focus=$(JOB) -skip=Conformance -nodes=1 -noColor=false ./... -- \
+	$(REPO_ROOT)/$(GINKGO_V1) -v -trace -tags=e2e -focus=$(JOB) -skip=Conformance -skipPackage=helpers -nodes=1 -noColor=false ./... -- \
```

Comment:
New e2e helpers package contains unit tests, but it doesn't run properly under the e2e configuration, so we exclude it.